sentry/src/main/java/io/sentry/CombinedScopeView.java

```diff
@@ -412,6 +412,13 @@ public void withTransaction(Scope.@NotNull IWithTransaction callback) {
     return globalScope.getSession();
   }
 
+  @Override
+  public void clearSession() {
```

Comment:
this is something i'm not entirely sure about

Reply:
In most places we simply use `getDefaultWriteScope()`. See the remove* methods.